test/streams.js

```diff
@@ -45,3 +48,87 @@ describe('strong-remoting', function() {
       .expect({bar: 'baz'}, done);
   });
 });
+
+describe('a function returning a ReadableStream', function() {
+  var Readable = require('stream').Readable;
+  var remotes = RemoteObjects.create();
+  var streamClass;
+  var server;
+
+  before(function(done) {
+    var test = this;
+    // NOTE: Date object will not be coerced back into a Date
+    var data = test.data = [{foo: 'bar'}, 'bat', false, 0, null];
```

Comment:
I see, could you please expand the comment to explain that the test data includes different object types, but that Date is intentionally excluded?

It would be good to document this limitation somewhere, or even emit a warning.